### PR TITLE
About page final

### PR DESCRIPTION
**Pull request by TheoBoschetti**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: frontend/src/Components/Game.jsx [hunk 1]
```diff
@@ -60,8 +60,8 @@ class Game extends React.Component {
 
   // Method which sets an event listener on keyboard inputs on all the screen as soon as the component mounts
   componentDidMount() {
-    this.gamepadMove();
-    this.getGamepad();
+    // this.getGamepad();
```
Comment (LoicBrassart) [inline frontend/src/Components/Game.jsx]: Nope

File: frontend/src/Components/Game.jsx [hunk 2]
```diff
@@ -73,6 +73,14 @@ class Game extends React.Component {
...
       this.attack(event);
     };
+
+    // if (this.state.canMove && this.state.gampadConnected) {
+    //   this.setState({ canMove: false });
+    //   setTimeout(() => {
+    //     this.setState({ canMove: true });
+    //   }, 120);
+    //   this.gamepadMove();
+    // }
```
Comment (LoicBrassart) [inline frontend/src/Components/Game.jsx]: Nope nope nope

File: frontend/src/Components/Game.jsx [hunk 3]
```diff
@@ -90,20 +98,20 @@ class Game extends React.Component {
     easystar.setAcceptableTiles([0]);
 
     easystar.findPath(xNPC, yNPC, x, y, path => {
-      if (path.length > 0) {
+      if (path.length > 0 && path.length < 10) {
```
Comment (LoicBrassart) [inline frontend/src/Components/Game.jsx]: Nothing to do with the About page

File: frontend/src/Components/Game.jsx [hunk 4]
```diff
@@ -125,60 +133,55 @@ class Game extends React.Component {
...
+  //     if (this.state.canMove && this.state.gampadConnected) {
+  //       this.setState({ canMove: false });
+  //       setTimeout(() => {
+  //         this.setState({ canMove: true });
+  //       }, 120);
+  //     }
+  //   };
+  //   window.requestAnimationFrame(update);
+
+  // }
```
Comment (LoicBrassart) [inline frontend/src/Components/Game.jsx]: NOOOOOOOOOOOOOOOOOOOOO !!!!!!!!!!!!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
I'm sure @GautierMille didn't read through this.
We're on the last PRs on this project, but please keep up good practices

**Comment by ghost** [comment]
ok

**Review by ghost — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**